src/main/kotlin/com/dansplugins/factionsystem/command/MedievalFactionsCommand.kt

```diff
@@ -15,14 +15,14 @@ class MedievalFactionsCommand(private val plugin: MedievalFactions) : CommandExe
             sender.sendMessage(plugin.language["DeveloperList", plugin.description.authors.joinToString()])
             sender.sendMessage(plugin.language["WikiLink"])
             sender.sendMessage(plugin.language["CurrentLanguage", plugin.config.getString("language") ?: "en_US"])
-            sender.sendMessage(plugin.language["SupportedLanguageList", Locale.getAvailableLocales().filter {
-                try {
-                    ResourceBundle.getBundle("lang", it)
-                    true
-                } catch(_: MissingResourceException) {
-                    false
-                }
-            }.joinToString { it.toString() }])
+            // sender.sendMessage(plugin.language["SupportedLanguageList", Locale.getAvailableLocales().filter {
```

Comment:
Prefer removing code over commenting out in comitted code, the code is there in git history if you need it

Reply:
Sounds good, the commented-out code has been removed.